modules/pam_faillock/pam_faillock.8.xml

```diff
@@ -134,10 +134,15 @@
                  <option>conf=/path/to/config-file</option>
                </term>
                <listitem>
-                 <para>
+                 <para condition="without_vendordir">
                    Use another configuration file instead of the default
                    <filename>/etc/security/faillock.conf</filename>.
                  </para>
+		 <para condition="with_vendordir">
+                   Use another configuration file instead of the default
+                   <filename>/etc/security/faillock.conf</filename> or
+		   <filename>%vendordir%/security/faillock.conf</filename> if it does not exists.
```

Comment:
`instead of the default which is to use the file <filename>/etc/security/faillock.conf</filename> or, if that one is not present, the file <filename>%vendordir%/security/faillock.conf</filename>.`